src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName, isMarried } = person;
+  const SEX_FEMALE = 'f';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        {age && <p className="Person__age">{`I am ${age}`}</p>}
+        <p className="Person__partner">
+          {isMarried
+            ? `${partnerName} is my ${sex === SEX_FEMALE ? 'husband' : 'wife'}`
```

Comment:
Let's create a variable for husband/wife to avoid nesting ternary operators
![image](https://github.com/mate-academy/react_person/assets/96409563/ba80dcc4-70d7-43fd-be6e-82270f90861d)
```suggestion
            ? `${partnerName} is my ${partner}`
```

Reply:
change added 4 days ago, but I didn't sent them to check:(